### DISCUSSION TITLE
Update on the development branch

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#2231] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we have pushed an update to the [development](https://github.com/NVIDIA/TensorRT-LLM/tree/main) branch (and the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main)) this Sep 17, 2024.

This [update](https://github.com/NVIDIA/TensorRT-LLM/pull/2230) includes:

* Features
  - Supported BFloat16 for fused Gated MLP.
  - Supported calibration with offline dataset for the Python `LLM` class.
* Bug fixes
  - Fixed a typo in `tensorrt_llm/models/model_weights_loader.py`, thanks to the contribution from @wangkuiyi in #2152.
  - Fixed duplicated import module in `tensorrt_llm/runtime/generation.py`, thanks to the contribution from @lkm2835 in #2182.

Thanks,
The TensorRT-LLM Engineering Team

**Comment by hofong428** [comment]
Awesome!
